gen/armor.py

```diff
@@ -454,7 +460,7 @@ def get_artilery_group_distance_from_frontline(self, group):
 
     def get_valid_position_for_group(self, conflict_position, isplayer, combat_width, distance_from_frontline):
         i = 0
-        while i < 25: # 25 attempt for valid position
+        while i < 1000: # 25 attempt for valid position
```

Comment:
Comment is now out of date. `for _attempt in range(1000):` does the same thing and explains itself :)